apps/yapms/src/routes/app/+layout.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
 	import '$lib/styles/global.css';
+	import '$lib/styles/map.css';
```

Comment:
I don't think we need this. The font should be built directly into the map I think. Removing it didn't change anything for me.

Reply:
On Chrome when I remove this, the map reverts back to a Times New Roman style font.
![image](https://user-images.githubusercontent.com/42476312/230198047-43c49721-ddbd-4213-931d-5a98068f464f.png)

Reply:
Ahh, maybe it's because I have Roboto installed directly on my operating system.